tasks/pdr-status-check/schemas/input.json

```diff
@@ -17,9 +17,16 @@
       }
     },
     "failed": {
+      "description": "List of failed execution arns with reason",
       "type": "array",
       "items": {
-        "type": "string"
+        "description:": "failed",
```

Comment:
Either update this to be descriptive or remove.

Reply:
And can you update the description and items schema for 'running' and 'completed' please?